src/components/HeroHeader/HeroHeader.tsx

```diff
@@ -20,8 +20,8 @@ export default function HeroHeader() {
   if (!screenWidth) return null;
 
   return (
-    <div id="hero" className="pt-2">
+    <header id="hero" className="pt-2">
```

Comment:
@devKiratu With this change, we now have two `header` elements on the page on the desktop version. Can you change the `header` element in the `Navbar` component so we only end up with one `header` on the page?

Reply:
Nice catch! Fixed via [f91d94a](https://github.com/reactdeveloperske/reactdevske-website/pull/107/commits/f91d94af912d4e55ed5820e30d7c02c29f7fac7a)